hail/Makefile

```diff
@@ -247,7 +247,7 @@ GITHUB_RELEASE_URL = https://github.com/hail-is/hail/releases/new
 .PHONY: tag
 tag: check-tag pypi-deploy
 	git tag $(HAIL_PIP_VERSION) -m "Hail version $(HAIL_PIP_VERSION)"
-	git push https://github.com/hail-is/hail.git $(HAIL_PIP_VERSION)
+	git push git@github.com:hail-is/hail.git $(HAIL_PIP_VERSION)
```

Comment:
This would have broken deploy for me:

```
$ git clone git@github.com:hail-is/hail.git
...
Warning: Permanently added 'github.com,140.82.114.3' (RSA) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Reply:
Ah you don't use ssh with git?

Reply:
The reason I made this change is because it asked me for my github username and password during the script, and I was having trouble pasting in my github password from password maanger. Not that a big a deal though, could just make the "lower" change.

Reply:
yeah, I think there are 2 ways to do it, and committing to either one will break devs who do the other. Do we parameterize this maybe?

We could use a named remote, maybe? That would trade this issue for an initial configuration step.

Reply:
Yeah, that could work. Is there any standard remote name we tend to use? I usually call my personal fork origin and the real project "upstream".

Reply:
Hmmm, maybe instead of `DEPLOY_PROD=1` we can just do `DEPLOY_PROD=<remote_url>`

Reply:
oh, I like that idea

Reply:
though it wouldn't need to be a url, "upstream" would work here too, right?

Reply:
Yeah, anything that git can interpret as remote should be fine I'd think.

Reply:
if you make that change, I'm totally happy with this